features/yield-calculator/YieldCalculator.tsx

```diff
@@ -0,0 +1,120 @@
+import { Box, TextField, Button, Typography } from "@material-ui/core";
+import { FormEvent, useState } from "react";
+import styled from "styled-components";
+import EmpState from "../../containers/EmpState";
+
+const Container = styled.div`
+  padding: 1rem;
+  border: 1px solid #434343;
+`;
+
+const FormInput = styled.div`
+  margin-top: 20px;
+  margin-bottom: 20px;
+`;
+
+const MS_TO_S = 1000;
+const S_TO_DAYS = 60 * 60 * 24;
+const DAYS_TO_YEAR = 365;
+
+const YieldCalculator = () => {
+  const { empState } = EmpState.useContainer();
+  const [tokenPrice, setTokenPrice] = useState<number | null>(0.9575);
+
+  const expirationTimestamp = empState.expirationTimestamp;
+
+  const calculateDaysToExpiry = () => {
+    if (expirationTimestamp) {
+      const currentTimestamp = Math.round(Date.now() / MS_TO_S);
+      const secondsToExpiry = expirationTimestamp.toNumber() - currentTimestamp;
+
+      return Math.round(secondsToExpiry / S_TO_DAYS);
+    } else {
+      return null;
+    }
+  };
+  const [daysToExpiry, setDaysToExpiry] = useState<number>(
+    calculateDaysToExpiry() || 30
+  );
```

Comment:
High-level comment: the way we're working with daysToExpiry is a little confusing... are we planning on pre-filling it with a calculated value while letting the user change that value? Should we just calculate it for them to make the UI simpler and give them fewer toggles? Not super opinionated here -- just might make this feel a little less technical and more intuitive.

This also may be a WIP, so if so, no worries.

Reply:
I think we should let the user manually change the daysToExpiry so they can play around with the yield calc.

Reply:
I'll rename the 30 magic number to PLACEHOLDER_DAYS_TO_EXPIRY

Reply:
```
// TODO: If the user has not selected an EMP (or has not even connected their web3 provider yet),
// then the EMP's expirationTimestamp will clearly be unavailable. In order to support explorability,
// this is a placeholder `daysToExpiry` that will be the default value for the textfield if no EMP is selected yet.
const PLACEHOLDER_DAYS_TO_EXPIRY = 30;
```

Reply:
SGTM! Thanks for the comment!